python/tk_mari/project.py

```diff
@@ -94,7 +94,7 @@ def create_project(self, name, sg_publishes, channels_to_create, channels_to_imp
             raise TankError("Newly created project '%s' wasn't opened!" % name)
         
         # add metadata to the project so that we can track the context:
-        self.md_mgr.set_project_metadata(new_project, engine.context)        
+        self.md_mgr.set_project_metadata(new_project, engine.context, version=1)
```

Comment:
Why not just call `set_project_version()` instead?

Reply:
Oh, yes, we can do that too. Will update.